contributing.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to OpenDevin
+
+Welcome to the OpenDevin community! We appreciate your interest in contributing to our open-source project. This document provides guidelines for contributing and information on the different leads and discussion channels.
+
+## Code of Conduct
+
+OpenDevin is committed to providing a welcoming and inclusive environment for all contributors. By participating, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Contribution Areas
+
+### Core Communications
+
+- **Lead:** Junyang Lin, Binyuan Hui
```

Comment:
Is it typical to list specific people in the contributing doc? I haven't seen this before. My guess is this will be fluid